server/node.go

```diff
@@ -375,8 +375,8 @@ func (n *Node) gossipCapacities() {
 		}
 		// Unique gossip key per store.
 		keyMaxCapacity := gossip.KeyMaxAvailCapacityPrefix +
-			strconv.FormatInt(int64(storeDesc.Node.NodeID), 10) + "-" +
-			strconv.FormatInt(int64(storeDesc.StoreID), 10)
+			strconv.FormatInt(int64(storeDesc.Node.NodeID), 16) + "-" +
```

Comment:
I actually think it would be better to specify as decimal everywhere. The reason is all of our admin UI pages will show them as decimal as well. Easier to visually debug.